### PR TITLE
docs: update Pub/Sub publisher docstring example

### LINKED ISSUE #3920: Pub/Sub: ApiFutures.addCallback() deprecated in code example
https://github.com/googleapis/google-cloud-java/blob/7e635696cb6d04aca1aa81016c8c934d4daa80ce/google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java#L177

I [updated this method](https://github.com/googleapis/google-cloud-java/blob/7e635696cb6d04aca1aa81016c8c934d4daa80ce/google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/PublisherSnippets.java#L58) in `google-cloud-examples`, but it also needs to be updated in `google-cloud-client/google-cloud-pubsub`.

### PR DESCRIPTION
**Pull request by anguillanneuf**
Fixes #3920

Missed the docstring in a previous PR #4120 that addressed the same issue.

### CODE REVIEW

**Comment by anguillanneuf** [comment]
@chingor13 Do you know why the test kept failing?

**Comment by chingor13** [comment]
It looks like a flakey spanner JDBC test. @kolea2, @olavloite might know more.

**Comment by kolea2** [comment]
Sorry about that, we're tracking #6075 to fix this. I've rerun the build.

File: google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google LLC
+ * Copyright 2019 Google LLC
```
Comment (kolea2) [inline google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java]: do we need to update the year @chingor13?

Comment (chingor13) [reply google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java]: This should stay 2016 - it's the year the file was first created/copyrighted

Comment (anguillanneuf) [reply google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java]: Thank you @kolea2 @chingor13. I will change it back to 2016.

**Pull request merged**